src/modules/sync/lib/environment-utils.ts

```diff
@@ -2,16 +2,14 @@ import { __ } from '@wordpress/i18n';
 import { SyncSite } from 'src/hooks/use-fetch-wpcom-sites/types';
 
 export const getSiteEnvironment = ( connectedSite: SyncSite ): string => {
-	if ( connectedSite.isPressable ) {
-		return connectedSite.environmentType ?? 'production';
-	}
-	return connectedSite.isStaging ? 'staging' : 'production';
+	return connectedSite.environmentType ?? 'production';
 };
 
 export const getEnvironmentLabel = ( type: string ): string => {
 	const labels: Record< string, string > = {
 		staging: __( 'Staging' ),
-		sandbox: __( 'Sandbox' ),
+		development: __( 'Development' ),
+		local: __( 'Local' ),
 		production: __( 'Production' ),
```

Comment:
We are now supporting all four environment types provided by core: https://developer.wordpress.org/reference/functions/wp_get_environment_type/.